addons/mail/static/src/models/message_reactions_summary/message_reactions_summary.js

```diff
@@ -0,0 +1,80 @@
+/** @odoo-module **/
+
+import { registerNewModel } from '@mail/model/model_core';
+import { attr, many2many, one2many, one2one } from '@mail/model/model_field';
+import { clear, create, insertAndReplace, link, replace, unlink } from '@mail/model/model_field_command';
+
+function factory(dependencies) {
+
+    class MessageReactionsSummary extends dependencies['mail.model'] {
+
+        //----------------------------------------------------------------------
+        // Public
+        //----------------------------------------------------------------------
+
+
+        //----------------------------------------------------------------------
+        // Private
+        //----------------------------------------------------------------------
+
+        _computeMessage() {
+            if (this.messageId) {
+                return insertAndReplace({
+                    id: this.messageId,
+                });
+            }
+            else {
+                return clear();
+            }
+        }
+
+        _computePartnersWhoReacted() {
+            let partners = [];
+            for (const partner of this.highlightedReaction[0].partners) {
+                partner.avatar = `/mail/channel/${this.message.originThread.id}/partner/${partner.id}/avatar_128`;
+                partners.push(partner);
+            }
+            for (const guest of this.highlightedReaction[0].guests) {
+                guest.avatar = `/mail/channel/${this.message.originThread.id}/partner/${guest.id}/avatar_128`;
+                partners.push(guest);
+            }
+            return partners;
+        }
+
+        //----------------------------------------------------------------------
+        // Handlers
+        //----------------------------------------------------------------------
+
+        /**
+         * @private
+         * @param {Object} messageReactionGroup
+         */
+        _onClickReaction(messageReactionGroup) {
+            this.update({
+                highlightedReaction: replace(messageReactionGroup)
+            });
+        }
+    }
+
+    MessageReactionsSummary.fields = {
+        /**
+         * States the OWL component of this message reactions summary.
+         */
+        component: attr(),
+        highlightedReaction: one2many('mail.message_reaction_group'),
+        message: one2one('mail.message', {
+            inverse: 'messageReactionsSummary',
+            readonly: true,
+        }),
+        partnersWhoReacted: attr({
+            compute: '_computePartnersWhoReacted',
+        }),
+    };
+
+    MessageReactionsSummary.identifyingFields = ['message'];
```

Comment:
ok as a data model, but the view model should always have the "parent view" as extra identifying field